### DISCUSSION TITLE
XYConf - Learn X in Y minutes unconference.

### DISCUSSION THREAD

**Discussion by narze** [creatorsgarten/discussions#9] [Event ideas]
Every attendees will have a chance to teach anything in 5-10 minutes per session, format can be Barcamp-ish with voting board, or pre-selected before the event.

Not sure if we should limit to development/coding areas or not. Maybe we'll see `Learn how to make หมูกรอบ in 5 minutes` if we don't.

**Comment by dtinth** [comment]
10 minute barcamps sounds fun!
